### ISSUE TITLE
version error(if used with win32)

### ISSUE THREAD

**Issue by kirill-21** [timsneath/filepicker_windows#19]
Because no versions of filepicker_windows match >2.0.1 <3.0.0 and filepicker_windows 2.0.1 depends on ffi ^1.0.0, filepicker_windows ^2.0.1 requires ffi ^1.0.0.
And because win32 >=2.7.0 depends on ffi ^2.0.0, filepicker_windows ^2.0.1 is incompatible with win32 >=2.7.0.
So, because exl_installer depends on both win32 ^2.7.0 and filepicker_windows ^2.0.1, version solving failed.
pub get failed (1; So, because exl_installer depends on both win32 ^2.7.0 and filepicker_windows ^2.0.1, version solving failed.)

**Comment by timsneath** [comment]
Thanks -- I've published 2.0.2 which expands the version window. Try a `flutter pub upgrade` (or a `dart pub upgrade`) and you should be all set.

**Comment by genment** [comment]
Hi, the issue raises again since new versions of win32 released

```
Because filepicker_windows 2.0.2 depends on win32 ^2.3.8 and no versions of filepicker_windows match >2.0.2 <3.0.0, filepicker_windows ^2.0.2 requires win32 ^2.3.8.
So, because my_app depends on both filepicker_windows ^2.0.2 and win32 ^4.1.0, version solving failed.
pub get failed (1; So, because hanhua_app depends on both filepicker_windows ^2.0.2 and win32 ^4.1.0, version solving failed.)
```
